FWCore/Common/interface/ProcessBlockHelper.h

```diff
@@ -0,0 +1,120 @@
+#ifndef FWCore_Common_ProcessBlockHelper_h
+#define FWCore_Common_ProcessBlockHelper_h
+
+/** \class edm::ProcessBlockHelper
+
+\author W. David Dagenhart, created 15 September, 2020
+
+*/
+
+#include "DataFormats/Provenance/interface/ProvenanceFwd.h"
+#include "FWCore/Common/interface/ProcessBlockHelperBase.h"
+
+#include <set>
+#include <string>
+#include <vector>
+
+namespace edm {
+
+  class ProcessBlockHelper : public ProcessBlockHelperBase {
+  public:
+    ProcessBlockHelperBase const* topProcessBlockHelper() const override;
```

Comment:
Would `final` be more appropriate?

Reply:
Done. I changed override to final. I am not sure which is in principle better. I'm not really expecting anyone to inherit from this class so it seems it does not matter. If someone did, I'm not sure if I would want to prevent them from overriding these functions or not.